contrib/for-tests/volumes/gluster/glusterd.vol

```diff
@@ -0,0 +1,10 @@
+volume management
+    type mgmt/glusterd
+    option working-directory /var/lib/glusterd
+    option transport-type socket,rdma
```

Comment:
Please add comments to clarify why these options are required, or remove the options if they're not required.